src/clr-angular/data/datagrid/datagrid-row-detail.ts

```diff
@@ -97,7 +110,12 @@ export class ClrDatagridRowDetail<T = any> implements AfterContentInit, OnDestro
     });
   }
 
+  // toggleExpand() {
```

Comment:
You can just remove that if we don't use it. You introduced it in this topic branch, so it's not a breaking change. 😉